app/controllers/api/v1/projects_controller.rb

```diff
@@ -0,0 +1,57 @@
+class Api::V1::ProjectsController < API::APIController
+
+  skip_before_filter :verify_authenticity_token
+
+  # GET /api/v1/projects
+  def index
+    @projects = Project.all
+    authorize! :manage, @projects
+    render json: {projects: @projects}
+  end
+
+  # GET /api/v1/projects/1.json
+  def show
+    begin
+      @project = Project.find(params[:id])
+      authorize! :manage, @project
+      render json: {project: @project}, status: 200
+    rescue ActiveRecord::RecordNotFound
+      render json: {error: "Project not found"}, status: 404
+    end
+  end
+
+  # POST /api/v1/projects
+  def create
+    @project = Project.new(params[:project])
+    authorize! :update, @project
```

Comment:
This should work, but I am wondering if we should use authorize! :create, @project instead?